modules/src/lib.rs

```diff
@@ -46,8 +46,8 @@ pub mod context_mock;
 #[cfg(test)]
 pub mod mock_client;
 
-/// Height of a block, same as in `tendermint` crate
-pub type Height = tendermint::block::Height;
+/// short use name for IBC Height domain type
```

Comment:
Great!

Reply:
I guess we should usually prefer:

```
use crate::Height
```
instead of

```
use crate::ics02_client::height::Height;
```
as often as possible.

Reply:
6b18f9f